cartoframes/data/registry/base_dataset.py

```diff
@@ -117,7 +117,7 @@ def is_public(self):
 
     def _create_context(self):
         if self._credentials:
-            return context.create_context(self._credentials.creds, self._credentials.session)
+            return context.create_context(self._credentials.creds)
```

Comment:
I don't understand this... are both `_credentials` and `creds` a Credentials object?

Reply:
I think it's a rename issue (from the old context to _credentials)

Reply:
Sorry, this is a bit of confusion from the [strategy pattern + factory in Dataset ](https://github.com/CartoDB/cartoframes/pull/834), let me explain. Now we have 2 "context":
- cartoframes.auth.Context (the classical one)
- cartoframes.context (the carto-python wrapper)

So, in order to avoid naming colisions, I change the `cartoframes.auth.Context` to `credentials` in Dataset.

This will be "fixed" in the next PR

Reply:
👌